daemon/provider/musicvideoinfo/spotifyprovider.py

```diff
@@ -173,3 +174,46 @@ def get_tracks_by_album_id(self, album_id):
                 break
 
         return tracks
+
+	def getNewRelease(self, offset_count, limit):
```

Comment:
The implemented name: `getNewRelease` is not equal to the interface name: `get_new_release_by_album_id`.